client/executor/common/src/sandbox/wasmer_backend.rs

```diff
@@ -100,14 +100,60 @@ pub fn invoke(
 	}
 }
 
+#[cfg(feature = "wasmer-cache")]
+enum CachedModuleErr {
+	FileSystemErr,
+	ModuleLoadErr(FileSystemCache, Hash),
+}
+
+#[cfg(feature = "wasmer-cache")]
+use CachedModuleErr::*;
+
+#[cfg(feature = "wasmer-cache")]
+fn get_cached_module(
+	wasm: &[u8],
+	store: &wasmer::Store,
+) -> core::result::Result<Module, CachedModuleErr> {
+	let fs_cache =
+		FileSystemCache::new("/tmp/substrate-wasmer-cache").map_err(|_| FileSystemErr)?;
+	let code_hash = Hash::generate(wasm);
+	unsafe { fs_cache.load(store, code_hash).map_err(|_| ModuleLoadErr(fs_cache, code_hash)) }
+}
+
+#[cfg(feature = "wasmer-cache")]
+fn try_to_store_module_in_cache(mut fs_cache: FileSystemCache, code_hash: Hash, module: &Module) {
+	let res = fs_cache.store(code_hash, &module.clone());
+	log::debug!("Store module cache with result: {:?}", res);
```

Comment:
I think you can remove these log messages.

Reply:
What?? why?

Reply:
We may wanna to reuse compiled cache even after node is stopped

Reply:
Make it as trace